source/fundamentals/linq.txt

```diff
@@ -575,3 +582,75 @@ implementation of LINQ:
 
    * - ``Where``
      - Returns all documents that match your specified criteria
+
+View Translated Queries
+-----------------------
+
+When you run a LINQ query, the {+driver-short+} automatically translates your
+query into an aggregation pipeline written with the {+query-api+}. You can view
```

Comment:
This currently reads "written with the MongoDB Query API" but the paragraph refers to translation, which to me implies another language. Would this be better as "written in the MongoDB Query Language (MQL)"?

Reply:
We don't refer to MQL as MQL or MongoDB Query Language externally anymore. It's been changed to the MongoDB Query API